docs/install_guides/installing-openvino-windows.md

```diff
@@ -111,104 +100,51 @@ You must update several environment variables before you can compile and run Ope
 
 > **NOTE**: If you see an error indicating Python is not installed when you know you installed it, your computer might not be able to find the program. For the instructions to add Python to your system environment variables, see <a href="#Update-Path">Update Your Windows Environment Variables</a>.
 
-The environment variables are set. Next, you will configure the Model Optimizer.
-
-## <a name="model-optimizer">Step 4: Configure the Model Optimizer
-
-The Model Optimizer is a Python\*-based command line tool for importing
-trained models from popular deep learning frameworks such as Caffe\*,
-TensorFlow\*, Apache MXNet\*, ONNX\* and Kaldi\*.
-
-The Model Optimizer is a key component of the Intel Distribution of OpenVINO toolkit. Performing inference on a model 
-(with the exception of ONNX and nGraph models) requires running the model through the Model Optimizer. When you convert a pre-trained 
-model through the Model Optimizer, your output is an Intermediate Representation (IR) of the network. The Intermediate 
-Representation is a pair of files that describe the whole model:
-
-- `.xml`: Describes the network topology
-- `.bin`: Contains the weights and biases binary data
-
-For more information about the Model Optimizer, refer to the [Model Optimizer Developer Guide](../MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md). 
+The environment variables are set. Next, you can download some additional tools.
 
-If you see error messages, make sure you installed all dependencies. These steps use a command prompt to make sure you see error messages.
+## <a name="model-optimizer">Step 3 (Optional): Download additional components
 
-1. Open a command prompt by typing `cmd` in your **Search Windows** box and then pressing **Enter**.
-Type commands in the opened window:
-   ![](../img/command_prompt.PNG)
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. These tools are now only available on [pypi.org](https://pypi.org/project/openvino-dev/).
 
-2. Go to the Model Optimizer prerequisites directory.<br>
-   ```sh
-   cd C:\Program Files (x86)\Intel\openvino_2021\deployment_tools\model_optimizer\install_prerequisites
-   ```
+Among the developer tools you can find e.g. Model Optimizer which imports, converts, and optimizes models, and Model Downloader which gives you access to the collection of pre-trained deep learning public and intel-trained models.  
```

Comment:
I believe we should add instruction or reference on how to install  `Install OpenVINO Developer Tools `
* (Recommended) Install manually by using the provided requirements files 
* Install from PyPI repository
Please sync up with  @xu-yuan1 proposal for this (apt & yum install guides)

Reply:
for now I added placeholder which should be removed once @xu-yuan1 adds the dedicated page